packages/sdk-auth/test/client-password-flow.spec.js

```diff
@@ -40,9 +44,12 @@ describe('Client Password flow', () => {
       .post(
         `/oauth/token`,
         // expected body
-        `grant_type=password&scope=manage_project:${config.projectKey}` +
-        `&username=user%204%5El*aJ%40ETso%2B%2F%5CHdE1!x0u4q5` + // encoded
-          `&password=pass%204%5El*aJ%40ETso%2B%2F%5CHdE1!x0u4q5` // encoded
+        encode({
```

Comment:
Also it may make the test more readable you are testing the encoding with the function to encode itself. I'm pretty sure that this works correct but a non working the encode function would result in a still working test.

But I also can see that the encode function may not produce a canonical string to compare it to a static string.

Reply:
Good point. Let me suggest something later :)

Reply:
To be on the safe side you could use https://nodejs.org/api/querystring.html#querystring_querystring_encode

Reply:
Yet another good point 💋. Thanks for being persistent. Here we go: 2913cfe.

Reply:
I take the thumbs up as an approval.